meta-flutter/README.md

```diff
@@ -0,0 +1,35 @@
+# meta-flutter for Yocto Project
+
+Recipe file examples for Yocto Project Yocto Project. See also: https://docs.yoctoproject.org/
+
+## Building Flutter
+
+### Wayland backend 
+```Shell
+$ bitbake flutter-client
+```
+
+### DRM-GBM backend
+You need to install the libsytemd to build this back end. Please add systemd into your conf/local.conf:    
+```
+DESTRO_FEATURES_append = " systemd"
+```
+See also: https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#using-systemd-for-the-main-image-and-using-sysvinit-for-the-rescue-image
+
+```Shell
+$ bitbake flutter-drm-backend
+```
+
+### DRM-EGLStream backend
+You need to install the libsytemd to build this back end. Please add systemd into your conf/local.conf:    
+```
+DESTRO_FEATURES_append = " systemd"
+```
+See also: https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#using-systemd-for-the-main-image-and-using-sysvinit-for-the-rescue-image
```

Comment:
You need to install `libsystemd` in the same way with the DRM-GBM backend.

Reply:
I fixed it.